cognite/neat/_graph/extractors/_classic_cdf/_classic.py

```diff
@@ -109,6 +109,18 @@ def __init__(
         self._target_external_ids_by_type: dict[InstanceIdPrefix, set[str]] = defaultdict(set)
         self._labels: set[str] = set()
         self._data_set_ids: set[int] = set()
+        self._extracted_labels = False
+        self._extracted_data_sets = False
+
+    def _get_activity_names(self) -> list[str]:
```

Comment:
what is an activity ?
Are these all the steps/actions == activities that will be executed by this extractor?
or all the activities that took place?